lib/team_api/canonicalizer.rb

```diff
@@ -9,8 +9,12 @@ class Canonicalizer
     # Canonicalizes the order and names of certain fields within site_data.
     def self.canonicalize_data(site_data)
       sort_collections site_data
```

Comment:
When it comes to multi-line calls, parentheses are required, or so I thought. But in the case of single-line calls, I find the parentheses-less version much easier to read, and prefer that style in the code I write.